doc/api.yml

```diff
@@ -222,6 +222,121 @@ paths:
               schema:
                 $ref: "#/components/schemas/User"
 
+  "/users/{uid}/pictures":
```

Comment:
I think that this endpoint can only be replace by the associated user (similar to getSelfFollowers) but more flexible in this way, make sure it is thoroughly tested in your implementation (security)

Reply:
yes